.github/workflows/update-release-info/action.yml

```
@@ -0,0 +1,39 @@
+name: Update Release Info
+description: Update current release version on external sites
+
+inputs:
+  repository:
+    type: string
+    default: disorderedmaterials/GudPy
+  createPR:
+    type: boolean
+    default: true
+  releaseVersion:
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+
+  - name: Checkout Develop
+    uses: actions/checkout@v3
+    with:
+      ref: develop
+
+  - name: Update Release Information
+    shell: bash
+    run: ./changeversion release -s ${{ inputs.releaseVersion }}
+
+
+  - name: Create Update PR
+    if: ${{ inputs.createPR == 'true' }}
+    uses: peter-evans/create-pull-request@v4
```

I noticed this just yesterday, but v4 is now old / outdated:
```suggestion
    uses: peter-evans/create-pull-request@v6
```